src/vstest.console/Processors/CollectArgumentProcessor.cs

```diff
@@ -123,15 +123,20 @@ public ArgumentProcessorResult Execute()
             return ArgumentProcessorResult.Success;
         }
 
-        internal static void EnableDataCollectorUsingFriendlyName(string argument, DataCollectionRunSettings dataCollectionRunSettings)
+        internal static void EnableDataCollectorUsingFriendlyName(string friendlyName, DataCollectionRunSettings dataCollectionRunSettings, string uri = null)
```

Comment:
If `Uri` is a required parameter (otherwise TMI will fail), should we allow a default null?

Reply:
Uri should be the second parameter for readability.